streams/src/main/java/org/apache/kafka/streams/processor/internals/ChangelogReader.java

```diff
@@ -56,4 +56,8 @@
      */
     void clear();
 
+    /**
+     * @return whether the changelog reader has been cleared and is empty.
```

Comment:
Hm, true. I was struggling to phrase this because technically it doesn't make sense for the `ChangelogReader` interface to be "empty" since it's not a data structure -- the specific `StoreChangelogReader` implementation just happens to keep (a lot of) state.
How about "...reader has just been cleared or is uninitialized"?